cpp/src/Slice/GrammarUtil.h

```diff
@@ -10,189 +10,79 @@
 
 namespace Slice
 {
-    class StringTok;
-    class StringListTok;
-    class TypeStringTok;
-    class TypeStringListTok;
-    class BoolTok;
-    class IntegerTok;
-    class FloatingTok;
-    class ExceptionListTok;
-    class ClassListTok;
-    class InterfaceListTok;
-    class EnumeratorListTok;
-    class ConstDefTok;
-    class OptionalDefTok;
-    class ClassIdTok;
```

Comment:
Is there any black-magic-C++ reason to forward declare all these super-simple classes?